src/Module/Inbox.php

```diff
@@ -25,19 +24,6 @@ public static function rawContent()
 			System::httpExit(400);
 		}
 
-// Enable for test purposes
```

Comment:
Please keep these lines - I need them from time to time.

Reply:
Either it's a full-on `@TODO` and I respect them, or it's just a test snippet and it would have a better place somewhere else on your system only.

Reply:
What do you mean with "it's a full-on @TODO"?

This code here is needed to save incoming raw data whenever there is the need to debug the processing of incoming stuff - which will be needed for the future as well, since it can always happen that there are incompatibilities with new implementations.

Reply:
Either it's necessary future code for a Friendica feature, or it's just a temporary change for debugging purpose that doesn't have its place in the main repo.

Reply:
I have to easily activate and deactivate these lines from time to time. Otherwise it's just blocking me.

Reply:
Please leave it on a local dev branch/stash on your machine if you need it frequently. No one else does.

Reply:
What about a config setting for this?

Reply:
There is a `system.debugging` that you can use for this, and a `system.loglevel` you can use to prevent hard drive flooding. I suggest you use the most verbose log level for this `debug`.

Reply:
No. For being able to resend the received content, it needs to be stored in single files that are machine readable.

Reply:
I get that, I'm just suggesting to check BOTH for `system.debugging == true` AND `system.loglevel == 'debug'`.

Reply:
It is especially needed for cases like "why don't we receive contact from AP implementation XYZ". So we should be able to activate and deactivate it independently from other settings. Having it activated just as a side effect because these loglevels had been set for some other reason would flood the hard drive indeed, since there (of course) is no auto cleaning.

Reply:
Make it so.

Reply:
It has been made.